bundles/statistics/statsgrid2016/instance.js

```diff
@@ -47,15 +50,24 @@ Oskari.clazz.define(
             // disable tile if we don't have anything to show or enable if we do
             // setup initial state
             this.setState();
+            
+            me.getTile().getFlyoutManager().init();
```

Comment:
How big of an operation would it be to move flyoutmanager to be in the instance? Since it's called a lot from here. Maybe we should refactor this so tile only handles the ui state of the tile, but the actual flyout handling would be handled in the instance.